staff/lab/ocf-tv

```diff
@@ -43,15 +43,25 @@ def wait_for_port(host, port, timeout=5, ssh_proc=None):
 
 
 def volume(args):
-    call(('ssh', args.host,
-          'export PULSE_SERVER=localhost; pactl set-sink-volume {} {}%'.format(args.sink, args.amount)))
+    cmd = 'export PULSE_SERVER=localhost; '
+
+    if args.amount is not None:
+        cmd += 'pactl set-sink-volume 0 {}%; '.format(args.amount)
+
+    # awk prints the 9th line after the match
+    cmd += "pactl list sinks | awk '/Sink #0/{nr[NR+9]}; NR in nr' | cut -d' ' -f 6;"
```

Comment:
Oh, just the way pactl prints the output, the volume is listed on the 9th line after the Sink name. I'll document that.

https://i.fluffy.cc/p1tmXBC7KfLCJwG2FvRpcrbD31HkKLrt.png